docs/development/core/development-functional-tests.asciidoc

```diff
@@ -8,17 +8,59 @@ We use functional tests to make sure the Kibana UI works as expected. It replace
 
 The `FunctionalTestRunner` is very bare bones and gets most of its functionality from its config file, located at {blob}test/functional/config.js[test/functional/config.js]. If you’re writing a plugin you will have your own config file. See <<development-plugin-functional-tests>> for more info.
 
-Execute the `FunctionalTestRunner`'s script with node.js to run the tests with Kibana's default configuration:
+
+The following will start Kibana, Elasticsearch and the chromedriver for you. To run the functional UI tests use the following commands
```

Comment:
Perfect, I suspected something wasn't quite right. Thanks for clarifying and the suggestions @spalger!